modules/mono/editor/bindings_generator.cpp

```diff
@@ -3919,57 +3919,67 @@ void BindingsGenerator::_initialize() {
 }
 
 static String generate_all_glue_option = "--generate-mono-glue";
-
-static void handle_cmdline_options(String glue_dir_path) {
-	BindingsGenerator bindings_generator;
-	bindings_generator.set_log_print_enabled(true);
-
-	if (!bindings_generator.is_initialized()) {
-		ERR_PRINT("Failed to initialize the bindings generator");
-		return;
-	}
-
-	CRASH_COND(glue_dir_path.is_empty());
-
-	if (bindings_generator.generate_cs_api(glue_dir_path.path_join(API_SOLUTION_NAME)) != OK) {
-		ERR_PRINT(generate_all_glue_option + ": Failed to generate the C# API.");
-	}
-}
+bool BindingsGenerator::generating_glue;
+String BindingsGenerator::glue_dir_path;
 
 static void cleanup_and_exit_godot() {
 	// Exit once done
 	Main::cleanup(true);
 	::exit(0);
 }
 
-void BindingsGenerator::handle_cmdline_args(const List<String> &p_cmdline_args) {
-	String glue_dir_path;
-
+void BindingsGenerator::process_cmdline(const List<String> &p_cmdline_args) {
```

Comment:
Why rename this method?

Reply:
No strong reason, it was mostly because I've been moving code around to split up the flag parsing from the binding generation, and ended up renaming this one somewhere in the process, and didn't rename it back.

Reply:
If there's no strong reason to rename it, I would prefer to keep the original name. In general, I recommend avoiding unnecessary changes.